test/cpp/end2end/xds/xds_fault_injection_end2end_test.cc

```diff
@@ -225,8 +225,8 @@ TEST_P(FaultInjectionTest, XdsFaultInjectionPercentageDelay) {
   auto* delay_percentage = http_fault.mutable_delay()->mutable_percentage();
   delay_percentage->set_numerator(kDelayPercentagePerHundred);
   delay_percentage->set_denominator(FractionalPercent::HUNDRED);
-  auto* fixed_delay = http_fault.mutable_delay()->mutable_fixed_delay();
-  fixed_delay->set_seconds(kFixedDelaySeconds);
+  SetProtoDuration(grpc_core::Duration::Seconds(kFixedDelaySeconds),
```

Comment:
kFixedDelaySeconds in this test and in other tests in this file is set to 100seconds. If we scale that, it will go up to 500seconds under UBSAN. I suggest changing 100seconds to 20seconds. That should still give us the DEADLINE_EXCEEDED errors that we want

Reply:
Good point.  Fixed.

In the process, I also found a few more places where we needed to take the slowdown factor into account.